packages/zosjobs/src/DownloadJobs.ts

```diff
@@ -101,11 +101,15 @@ export class DownloadJobs {
         ImperativeExpect.keysToBeDefined(parms, ["jobFile"], "You must specify a job file on your 'parms' parameter" +
             " object to the downloadSpoolContentCommon API.");
 
-        const file = DownloadJobs.getSpoolDownloadFilePath(parms);
-        this.log.debug("Downloading spool file %s for job %s(%s) to file %s",
-            parms.jobFile.ddname, parms.jobFile.jobname, parms.jobFile.jobid, file);
-        IO.createDirsSyncFromFilePath(file);
-        IO.createFileSync(file);
+        this.log.debug(`Downloading spool file %s for job %s(%s)`,
+            parms.jobFile.ddname, parms.jobFile.jobname, parms.jobFile.jobid);
```

Comment:
When the `stream` param is null or undefined, should we still include the file path in the log message?